features/contract-state/Totals.tsx

```diff
@@ -96,7 +96,7 @@ const Totals = () => {
               <Small> {tokenSymbol}</Small>
             </Tooltip>
           </Typography>
-          <Tooltip title="This is the total number of tokens minted minus the total number of tokens redeemed.">
+          <Tooltip title="Total number of tokens minted minus the total number of tokens redeemed.">
```

Comment:
Two things here:

- I think its more straightforward to state that this is the total number of synthetic tokens outstanding. This is not calculated as: total-tokens-minted - total-tokens-burned, its much more simply calling `emp.totalTokensOutstanding()`.
- The tool tip is a bit hard to find and moreover it overlaps with the <DataBox> component. Can you try wrapping the <Tooltip> around the <DataBox> element?

![Screen Shot 2020-07-23 at 20 28 14](https://user-images.githubusercontent.com/9457025/88351034-09c9f000-cd23-11ea-8e45-1df32c5728a9.png)

Reply:
Ye, I didn't add the tooltip. I just didn't like the wording it originally had. I agree it can be improved a bunch. I rate we leave this for a separate PR where we can add tooltips in a bunch of different places. See issue #62

Reply:
For now I've stripped out these tooltips.